scripts/install_environment.sh

```diff
@@ -202,7 +202,12 @@ function additional_configs() {
     sudo chmod ugo+rx "$(dirname "$(pwd)")"
     echo "disabling selinux by setenforce 0"
     sudo setenforce 0 || true
-    sudo chmod 600 ssh_key/key
+
+    if [ ! -f ~/.ssh/id_rsa ]; then
+        ssh-keygen -t rsa -f ~/.ssh/id_rsa -P ''
```

Comment:
ssh-keygen always exists ? or maybe we should install it?

Reply:
it's from ``openssh`` which seems to always be installed